Writing to images by RS232 mounters

Rider1982 opened the pull request:
1. Support in Mounter.wmf
2. Updated both console and GUI mounters
3. Added GUI Mounter and vdos sources

Rider1982 committed:
Поддержка записи на образы RS232 Mounter:
1. Поддержка в Mounter.wmf
2. Обновлены консольный и GUI mounter'ы
3. Добавлены исходники RS232Mounter GUI и vdos

tslabs commented:
pentevo/tools/rs232mnt/rs232mnt/rs232mnt.cpp
- Fix bug in line 265: read data+crc in a single FIFO read operation.
- Add .trd update on disk. (Now mounter only updates .trd image in RAM.)
- Open .trd in exlusive mode to avoid simultaneous access from other apps.

Files:
- remove:
  pentevo\soft\WC\source\plugins\mounter\vdos_sjasmplus\emu.bin
  pentevo\soft\WC\source\plugins\mounter\vdos_sjasmplus\emu.bin.diff
  pentevo\soft\WC\source\plugins\mounter\vdos_sjasmplus\emu.bin.perfect

- zip 
  pentevo/soft/WC/source/plugins/mounter/vdos_sjasmplus/other/tasm412help_1.sna

Rider1982 committed:
Updates according to code review
1. Reading sector data and crc as a single chunk
2. Writing received data to a TRD image

Rider1982 committed:
Fix: lost CRC on 3.5+ MHz

Rider1982 committed:
Fix:rs232mnt doest'n crash on r/w errors while data excahge
